src/admin-views/editor/panel/list.php

```diff
@@ -191,6 +191,18 @@ class="ticket_panel panel_base"
 			<?php endif; ?>
 		</div>
 	</div>
+	<div class="tec_ticket-panel__helper_text__wrap">
+		<p>
+			<?php
+			$helper_link = sprintf(
+				'<a href="%1$s" target="_blank" rel="noopener noreferrer ">%2$s</a>',
+				esc_url( 'https://evnt.is/manage-tickets' ),
+				esc_html__( 'Learn more about ticket management', 'event-tickets' )
+			);
+			echo sprintf( esc_html__( 'Create and manage Series Passes for this Series. %s', 'event-tickets' ), $helper_link );
```

Comment:
This has to be filterable, probably an action, in which you would jump to customize the code for Series Passes from the FT project.

Reply:
I propose we use the `tribe_events_tickets_after_new_ticket_panel` action here to render this.
For RSVP and Tickets (the default type) hook in the ET provider, for Series Passes, hook in the FT provider.